WalkCraft.lua

```diff
@@ -198,6 +215,9 @@ end
 
 function WalkCraft.OnClick(self, aButton)
     if (aButton == "RightButton") then
```

Comment:
At some point we'll probably want to keep a key mapping file / enum

Reply:
ayayyay